### PR TITLE
Explain valid negative idle

### PR DESCRIPTION
**Pull request by michaelmdresser**

### CODE REVIEW

File: allocation.md [hunk 1]
```diff
@@ -157,6 +157,37 @@ Here is an example allocation for the `cost-model` container in a pod in Kubecos
 - `__unallocated__` refers to aggregated allocations without the selected `aggregate` field; e.g. aggregating by `label:app` might produce an `__unallocated__` allocation composed of allocations without the `app` label.
 - `__unmounted__` (or "Unmounted PVs") refers to the resources used by PersistentVolumes that aren't mounted to a Pod using a PVC, and thus cannot be allocated to a Pod.
 
+#### Negative Idle
+
+It is possible to have a negative idle cost that is technically valid in the following
+situation:
+
+- You have workloads that _request_ resources, but do not use all of them
+- You have workloads the _use_ resources beyond the scheduled capacity of the node
```
Comment (dwbrown2) [inline allocation.md:166]: These have to happen in conjunction, yeah? Maybe add **AND** to clarify if so?

File: allocation.md [hunk 2]
```diff
@@ -157,6 +157,37 @@ Here is an example allocation for the `cost-model` container in a pod in Kubecos
 - `__unallocated__` refers to aggregated allocations without the selected `aggregate` field; e.g. aggregating by `label:app` might produce an `__unallocated__` allocation composed of allocations without the `app` label.
 - `__unmounted__` (or "Unmounted PVs") refers to the resources used by PersistentVolumes that aren't mounted to a Pod using a PVC, and thus cannot be allocated to a Pod.
 
+#### Negative Idle
+
+It is possible to have a negative idle cost that is technically valid in the following
```
Comment (dwbrown2) [inline allocation.md:162]: Is it worth saying you should confirm with Diagnostics that you're collecting data correctly?

Comment (michaelmdresser) [reply allocation.md:164]: Added a note, what do you think?

File: allocation.md (hunk 1 above)
Comment (michaelmdresser) [reply allocation.md:168]: Done!

File: allocation.md (hunk 2 above)
Comment (dwbrown2) [reply allocation.md:164]: LGTM, but maybe add a link to Diagnostics docs if you have one that you like?

**Review by dwbrown2 — APPROVED** [review]

File: allocation.md (hunk 2 above)
Comment (michaelmdresser) [reply allocation.md:164]: Good idea, done.

**Pull request merged**